packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount-mobile.jsx

```diff
@@ -22,15 +22,21 @@ const Basis = observer(
         const { addToast } = ui;
         const { currency } = client;
         const {
+            contract_type,
             is_turbos,
             is_vanilla,
             onChangeMultiple,
+            stake_boundary,
             trade_amount,
             trade_basis,
             trade_duration_unit,
             trade_duration,
-            contract_type,
+            vanilla_trade_type,
```

Comment:
Definitely, we will have conflicts as `vanilla_trade_type` will be deprecated here: https://github.com/binary-com/deriv-app/pull/9858

Reply:
@kate-deriv sadly, it's inevitable, @henry-deriv can't apply your changes in his PR.

Reply:
I know, anyway someone will fix the conflict (me or Henry)